src/controllers/ContinuedIllnessController.ts

```diff
@@ -0,0 +1,77 @@
+import Joi from '@hapi/joi';
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { YesNo } from 'app/models/fields/YesNo';
+import { createSchema } from 'app/models/fields/YesNo.schema';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'continued-illness';
+
+const navigation : Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+@controller(CONTINUED_ILLNESS_PAGE_URI, IllnessReasonFeatureMiddleware, SessionMiddleware, AuthMiddleware)
+export class ContinuedIllnessController extends BaseController<any> {
+
+    constructor() {
+        const errorMessage = 'You must tell us if this is a continued illness';
+        const schema: Joi.AnySchema = Joi.object({
+            continuedIllness: createSchema(errorMessage)
+        }).unknown(true);
+
+        super(
+            template,
+            navigation,
+            new FormValidator(schema)
+        );
+    }
+
+    protected prepareViewModelFromAppeal(appeal: Appeal): any {
+        const illnessStartDateRaw = appeal.reasons.illness?.illnessStart;
+
+        if (!illnessStartDateRaw) {
```

Comment:
Initially entered this to surpass the type-checking (was throwing an error because the object may be undefined) but I've now found another way around this which doesn't involve error handling.

Reply:
`appeal.reasons.illness!.illnessStart`? Exclamation mark will silence type checking telling type checker that in this scenario start date will always be there.

Reply:
Great minds think alike, that's exactly what I've done.